From 4f941694919aaf6a45c8fe3195f719ad70b3409c Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Mon, 24 Feb 2014 22:15:07 -0800
Subject: ASoC: rsnd: remove verbose debug message from scu/ssi

scu/ssi probe() already have more detail debug message.

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Signed-off-by: Mark Brown <broonie@linaro.org>
(cherry picked from commit 98455ed5ddbfbcbcd0adb300c6cb1964ed0a6931)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 sound/soc/sh/rcar/scu.c | 1 -
 sound/soc/sh/rcar/ssi.c | 2 --
 2 files changed, 3 deletions(-)

diff --git a/sound/soc/sh/rcar/scu.c b/sound/soc/sh/rcar/scu.c
index 82d8b20a6ffb..882e837d39d1 100644
--- a/sound/soc/sh/rcar/scu.c
+++ b/sound/soc/sh/rcar/scu.c
@@ -663,7 +663,6 @@ int rsnd_scu_probe(struct platform_device *pdev,
 
 		dev_dbg(dev, "SCU%d probed\n", i);
 	}
-	dev_dbg(dev, "scu probed\n");
 
 	return 0;
 }
diff --git a/sound/soc/sh/rcar/ssi.c b/sound/soc/sh/rcar/ssi.c
index 2460c9f564de..38a62a883b83 100644
--- a/sound/soc/sh/rcar/ssi.c
+++ b/sound/soc/sh/rcar/ssi.c
@@ -589,8 +589,6 @@ int rsnd_ssi_probe(struct platform_device *pdev,
 		rsnd_ssi_parent_clk_setup(priv, ssi);
 	}
 
-	dev_dbg(dev, "ssi probed\n");
-
 	return 0;
 }
 
-- 
2.1.2

